lib/Units.pm

```diff
@@ -1,998 +1,789 @@
+		kg     => 1,
+		s      => -2
+	},
+	keV => {     # kilo electron volt
+		factor => 1.6022E-16,
+		m      => 2,
+		kg     => 1,
+		s      => -2
+	},
+	MeV => {     # mega electron volt
+		factor => 1.6022E-13,
+		m      => 2,
+		kg     => 1,
+		s      => -2
+	},
+	GeV => {     # giga electron volt
+		factor => 1.6022E-10,
+		m      => 2,
+		kg     => 1,
+		s      => -2
+	},
+	TeV => {     # tera electron volt
+		factor => 1.6022E-7,
+		m      => 2,
+		kg     => 1,
+		s      => -2
+	},
+	kWh => {     # kilo Watt hour
+		factor => 3.6E6,
+		m      => 2,
+		kg     => 1,
+		s      => -2
+	},
+
+	# POWER: fundamental unit m kg / s^3
+	W => {       # watt
+		factor => 1,
+		m      => 2,
+		kg     => 1,
+		s      => -3
+	},
+	kW => {      # kilowatt
+		factor => 1000,
+		m      => 2,
+		kg     => 1,
+		s      => -3
+	},
+	MW => {      # megawatt
+		factor => 1E6,
+		m      => 2,
+		kg     => 1,
+		s      => -3
+	},
+	mW => {      # milliwatt
+		factor => 0.001,
+		m      => 2,
+		kg     => 1,
+		s      => -3
+	},
+	hp => {      # horse power
+		factor => 746,
+		m      => 2,
+		kg     => 1,
+		s      => -3
 	},
+
 	# PRESSURE
-	# Pa     -- Pascal
-	# kPa    -- kilo Pascal
-	# MPa    -- mega Pascal
-	# GPa    -- giga Pascal
-	# atm    -- atmosphere
-	# bar	 -- 100 kilopascals
-	# cmH2O	 -- centimetres of water
-	#
-	'Pa' => {
-		'factor' => 1,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'kPa' => {
-		'factor' => 1000,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'MPa' => {
-		'factor' => 1E6,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'GPa' => {
-		'factor' => 1E9,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'atm' => {
-		'factor' => 1.01E5,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'bar' => {
-		'factor' => 100000,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'mbar' => {
-		'factor' => 100,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'Torr' => {
-		'factor' => 133.322,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'mmHg' => {
-		'factor' => 133.322,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'cmH2O' => {
-		'factor' => 98.0638,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
-	},
-	'psi' => {
-		'factor' => 6895,
-		'm'      => -1,
-		'kg'     => 1,
-		's'      => -2
+	Pa => {      # pascal
+		factor => 1,
+		m      => -1,
+		kg     => 1,
+		s      => -2
+	},
+	kPa => {     # kilopascal
+		factor => 1000,
+		m      => -1,
+		kg     => 1,
+		s      => -2
+	},
+	MPa => {     # megapascal
+		factor => 1E6,
+		m      => -1,
+		kg     => 1,
+		s      => -2
+	},
+	GPa => {     # gigapascal
+		factor => 1E9,
+		m      => -1,
+		kg     => 1,
+		s      => -2
+	},
+	atm => {     # atmosphere
+		factor => 1.01E5,
+		m      => -1,
+		kg     => 1,
+		s      => -2
+	},
+	bar => {
+		factor => 100000,
+		m      => -1,
+		kg     => 1,
+		s      => -2
+	},
+	mbar => {    # megabar
+		factor => 100,
```

Comment:
comment for `mbar` should be millibar, not megabit.  factor is correct.

Reply:
Thanks for looking this over and finding these things.  I have fixed that mistake.